lib/commands/dialog/index.js

```diff
@@ -32,7 +32,7 @@ class DialogCommand extends AbstractCommand {
     }
 
     optionalOptions() {
-        return ['skill-id', 'locale', 'stage', 'replay', 'profile', 'debug'];
+        return ['skill-id', 'locale', 'stage', 'replay', 'output', 'profile', 'debug'];
```

Comment:
IMO the option name is not very precise since now we do both input and output, in the web console this concept is termed as `skill I/O`, I personally think we can keep using this term. Now we might be able to call it `ask dialog --save-skill-io` what do you think?